src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -41,7 +42,7 @@
   private DatastoreService datastore;
 
   // API Key for Firebase
-  private static String API_ACCESS_KEY="AAAAQ8CY2j0:APA91bHKNT-HNdbxyO-eD671RuQlkZOgMS7VTz66uwtxVr8kDfUDcGCLojY1hgRsXK9IfyE1LmTYarOO6gq_4CThif_6bjgmvh6JIikM28HZTAQs-u7jtEkEUokDHrpWvZ9jsI6Bmfyl";
+  private static String API_ACCESS_KEY="AAAAQ8CY2j0:APA91bGOgQlivDmrkUFqN92TGTuVyooVXFPzggOAjPF79njQDhIE-JcT8UnEUBUJahlv_mcVNj-Yo2N6MTwvZkuIvqIq8vP_kMLT7HlZmyGPcLRLRAYSBfMUgz5D2PVzqXVaYG3Ff5zBmHomQXgAnsh_uqoWZG9Vzw";
```

Comment:
Is this meant to be secret?

Reply:
yes. Where can I store it to make it secret?

Reply:
I put it as an entity in datastore, but it's going to be hard to test.